### ISSUE TITLE
Flywheel PID characterization

### ISSUE THREAD

**Issue by omentic** [Spartronics4915/2020-InfiniteRecharge#5]
Builds off of #3

**Comment by pietroglyph** [comment]
[My PR](https://github.com/wpilibsuite/frc-characterization/pull/83) in frc-characterization needs to be merged before I teach @ethanrini (and maybe @Elkwizard?) how to do this.

**Comment by binnur** [comment]
I don't understand the dependency on formally merging on wpilib for this task. Process of teaching can also help augment the documentation needs, and I would suggest tagging @ethanrini to review and update existing documentation as needed.

**Comment by pietroglyph** [comment]
We depend on the merge because the merge happens when I finish testing the tool. I don’t want to be debugging while I teach.

**Comment by omentic** [comment]
The mechanics have finished the final launcher. Characterization likely will happen Tuesday / Friday. Creating the InterpolatingTreeMap needs it.

**Comment by dbadb** [comment]
I agree with @binnur that we should try to avoid dependencies on third parties that can hold up our ability to move forward.  Isn't there a time-lag between wpilib integration and actual deployment?  Isn't this harder to happen as the season progresses?

**Comment by pietroglyph** [comment]
I’m not really concerned about this type of third party dependency.

a. frc-characterization doesn’t have the same release cadence as wpilib. Whatever is in upstream gets run through a CI pipeline and deployed immediately to PyPI.
b. The PR was merged nearly a month ago.
c. This isn’t really a 3rd party dependency because I wrote it, and if we really wanted to run it without it being merged it’s not very hard.
